hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/process/Signal.java

```diff
@@ -47,7 +64,11 @@ SubscriptionName getTarget() {
     }
 
     boolean canExecuteNow(long currentTimestamp) {
-        return currentTimestamp > executeAfterTimestamp;
+        return currentTimestamp >= executeAfterTimestamp;
```

Comment:
In one place signal is created with clock.now() which means that it can executed in the same ms.